lib/cib/Makefile.am

```diff
@@ -1,5 +1,5 @@
 #
-# Copyright 2004-2018 the Pacemaker project contributors
+# Copyright 2004-2023 the Pacemaker project contributors
```

Comment:
I guess this is leftover from earlier version

Reply:
Yep, I started to make a `libcib_private.h` file and a `cib_file_transaction.c` file. Decided to leave that alone for now.